app/routes/eligible-organisations.js

```diff
@@ -1,11 +1,29 @@
-const getEligibility = require('../eligibility')
+const cache = require('../cache')
+// const getEligibility = require('../eligibility')
+const ViewModel = require('./models/search')
+const schema = require('./schemas/sbi')
 
 module.exports = [{
   method: 'GET',
   path: '/eligible-organisations',
   options: {
     handler: async (request, h) => {
-      const { eligibility } = await getEligibility(request)
+      // const { eligibility } = await getEligibility(request)
+
+      // temp statements
```

Comment:
This is temporary as we don't have the queues set up yet and it would remain stuck at this line if we didn't hardcode